src/components/01-branding/02-typography/heading/heading-default.njk

```diff
@@ -0,0 +1,78 @@
+<table class="usa-table">
```

Comment:
Please make reusable (see icons/lists/actions--single)  remove markup. If you want the configuration section it can go in notes